### ISSUE TITLE
Documentation related to virtual switch selection on Windows+Hyper-V

### ISSUE THREAD

**Issue by gbraad** [minishift/minishift#843]
The documentation for the virtual switch selection can currently be found in the installing.adoc. I do not really believe this is the right location, unless we also link to the Note from either the troubleshooting guide or the running instructions.

Also, an example for `cmd` and/or `powershell` is preferred as the command itself can lead to a possible error. The name of the switch should be specified as entered in the Hyper-V manager, and NOT as the actually created virtual switch. Note: created as `External (Wireless)`, will lead to a bridge called `vEthernet (External (Wireless))` in the network adapters. Below you can see the difference in how to specify the name.

cmd:
```cmd
C:\> set HYPERV_VIRTUAL_SWITCH=External (Wireless)
... and not
C:\> set HYPERV_VIRTUAL_SWITCH="External (Wireless)" 
```

Notice that using quotes will result in errors of not finding the virtual switch:

      Error creating the VM. Error with pre-create check: "vswitch \"\\\"External (Wireless)\\\"\" not found"

powershell:
```powershell
PS> $env:HYPERV_VIRTUAL_SWITCH="External (Wireless)"
```

Related to: #590

**Comment by Preeticp** [comment]
Considering that out of the 5 hypervisors Minishift supports, 2 are embedded in the Minishift and do not need additional installation steps(vbox and VM ware). Out of the other 3, KVM and Xhyve are covered in the Docker Machine Drivers doc. 
I suggest we expand the Docker Machine Drivers to include information about HyperV too. We can rename the topic to a generic - Installing Hypervisors. WDYT?

**Comment by gbraad** [comment]
Yes... Or something like: "Setup hypervisors drivers"...

**Comment by Preeticp** [comment]
Makes sense @gbraad

**Comment by hferentschik** [comment]
> We can rename the topic to a generic - Installing Hypervisors. WDYT?

-1 for a rename. We choose this name explicitly, since it is the most appropriate one and is also the term which when used for example with Google gets you to the most relevant documentation.

Also a "hypervisor driver" is imo quite ambiguous.

**Comment by hferentschik** [comment]
Merged via pull request #951

**Comment by prbhadra** [comment]
We did setup an environment variable and getting the following error.

We have no clue on where things are going wrong.

C:\Users\prbha>echo %HYPERV_VIRTUAL_SWITCH%
External (Wireless)

C:\Users\prbha>minishift start
-- Starting profile 'minishift'
-- Checking if requested OpenShift version 'v3.9.14' is valid ... OK
-- Checking if requested OpenShift version 'v3.9.14' is supported ... OK
-- Checking if requested hypervisor 'hyperv' is supported on this platform ... OK
-- Checking if Hyper-V driver is installed ... OK
-- Checking if Hyper-V driver is configured to use a Virtual Switch ...
   'External (Wireless)' ... OK
-- Checking if user is a member of the Hyper-V Administrators group ... OK
-- Checking the ISO URL ... OK
-- Checking if provided oc flags are supported ... OK
-- Starting local OpenShift cluster using 'hyperv' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    4 GB
   vCPUs :    2
   Disk size: 20 GB
-- Starting Minishift VM ...... FAIL E0520 07:01:13.124673    4368 start.go:368] Error starting the VM: Error creating the VM. Error with pre-create check: "vswitch \"External (Wireless)\" not found". Retrying.
Error starting the VM: Error creating the VM. Error with pre-create check: "vswitch \"External (Wireless)\" not found"
